pkg/api/defaults-kubelet_test.go

```diff
@@ -145,6 +145,17 @@ func TestKubeletConfigDefaultsRemovals(t *testing.T) {
 				key)
 		}
 	}
+
+	expected := map[string]string{
```

Comment:
let's also add a check to make sure that these values aren't set for k8s < 1.15.0-beta.1. You can use the cs above since it's 1.12 https://github.com/Azure/aks-engine/pull/1416/files#diff-c116fab93edb004accebd12e5742e80cR116.